ecr-login/ecr.go

```diff
@@ -115,13 +115,28 @@ func (self ECRHelper) Get(serverURL string) (string, string, error) {
 		return "", "", credentials.NewErrCredentialsNotFound()
 	}
 
+	profile, err := config.GetRegistryProfile(serverURL)
+	if err != nil {
+		self.logger.
+			WithError(err).
+			WithField("AWS profile", profile).
+			Errorf("Error fetching profile from config for repository %v", serverURL)
+		return "", "", credentials.NewErrCredentialsNotFound()
+	}
+
 	var client api.Client
 	if registry.FIPS {
 		client, err = self.clientFactory.NewClientWithFipsEndpoint(registry.Region)
```

Comment:
Is there a scenario where we want to explicit use an explicit profile for Fips endpoints?